model/github_runner.rb

```diff
@@ -63,14 +63,18 @@ def check_pulse(session:, previous_pulse:)
       available_memory = session[:ssh_session].exec!("awk '/MemAvailable/ {print $2}' /proc/meminfo").chomp
       "up"
     rescue
-      "down"
+      is_destroying? ? "up" : "down"
```

Comment:
Can't we just ignore it if it's destroying instead of saying it's `up`?

Reply:
I guess we can just return nil. Still, I don't know its impact on the monitoring. So, @byucesoy to the rescue.

Reply:
Monitor actually refreshes its model cache to remove resources it doesn't need to monitor, but it does this only every 5 minutes, because we want to limit database access as much as possible.

I guess eventual solution we would converge would be some sort of pub/sub mechanism to inform monitor about model changes. Though, it is a too big of a change at the moment and not a high priority.